File: TP2/Java/Punto 5/MORA/punto5/Persona.java
import java.time.LocalDate;

public class Persona {
    private String nombre;
    private String apellido;
    private LocalDate fechaNacimiento;

//constructor
    public Persona (String nombre, String apellido, LocalDate fechaNacimiento){ 
        this.nombre = nombre;
        this.apellido= apellido;
        this.fechaNacimiento = fechaNacimiento;
    }

    //getters 
    public String getNombre (){
        return nombre;
    }
    public String getApellido (){
        return apellido;
    }
    public LocalDate getFechaNac (){
        return fechaNacimiento;
    }


    //setters
    public void setNombre(String nombre){
        this.nombre = nombre;
    }
    public void setApellido(String apellido){
        this.apellido = apellido;
    }
    public void setFechaNac(LocalDate fechaNacimiento){
        this.fechaNacimiento = fechaNacimiento;
    }
    
    @Override
    public String toString(){
        return "//Nombre: " + nombre + "// Apellido: " + apellido + "// Fecha de nacimiento " + fechaNacimiento;

    }
}

File: TP2/Python/Punto5/cesar/Persona.py

class Persona:
    def __init__(self,nombre,apellido,fecha_nacimiento):
        self.__nombre = nombre
        self.__apellido = apellido
        self.__fecha_nacimiento = fecha_nacimiento
    
    @property
    def nombre(self):
        return self.__nombre
    @nombre.setter
    def nombre(self,nombre):
        self.__nombre = nombre
    @property
    def fecha_nacimiento(self):
        return self.__fecha_nacimiento

    @property
    def apellido(self):
        return self.__apellido 

    @apellido.setter
    def apellido(self,apellido):
        self.__apellido = apellido

    @fecha_nacimiento.setter
    def FechaNacimiento(self, fecha_nacimiento):
        self.__fecha_nacimiento = fecha_nacimiento

    def toString(self):
        print("Nombre: {} Apellido: {} Fecha de nacimiento: {}".format(self.__nombre,self.__apellido,self.__fecha_nacimiento))
